modular_skyrat/modules/importedvendors/code/vendors.dm

```diff
@@ -0,0 +1,182 @@
+/obj/effect/spawner/random/vending/snackvend
+	loot = list(
+		/obj/machinery/vending/imported,
+		///obj/machinery/vending/imported/panslav,
```

Comment:
I'd leave a comment saying that this is in a near-future PR, or simply remove this until it's added - it's all modular, you won't have to worry about someone overwriting it all anyways